examples/gno.land/p/demo/audio/bytebeat/bytebeat.gno

```diff
@@ -0,0 +1,70 @@
+package bytebeat
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"math"
+
+	"gno.land/p/demo/audio/biquad"
+	"gno.land/p/demo/audio/wav"
+	"gno.land/p/demo/ufmt"
+)
+
+func ByteBeat(seconds uint32, sampleRate uint32, bytebeat_func func(t int) int) (data string) {
+	b := new(bytes.Buffer)
+	w := bufio.NewWriter(b)
+
+	var numSamples uint32 = sampleRate * seconds / 2
+	var numChannels uint16 = 1
+	var bitsPerSample uint16 = 16
+
+	writer, err := wav.NewWriter(w, numSamples*2, numChannels, sampleRate, bitsPerSample)
+	if err != nil {
+		data = ufmt.Sprintf("error: %s", err)
+		return
+	}
+
+	samples := make([]wav.Sample, numSamples)
+	maxValue := 0
+	lastValue := 0
+	for i := int(0); i < int(numSamples); i++ {
+		lastValue = bytebeat_func(i)
+		samples[i].Values[0] = lastValue
+		if samples[i].Values[0] > maxValue {
+			maxValue = samples[i].Values[0]
+		}
```

Comment:
remove lastValue, and 

```suggestion
		pitch := bytebeat_func(i)
		samples[i].Values[0] = pitch
		if pitch > maxValue {
			maxValue = pitch
		}
```

can bytebeat_func return negative values?

if so, then maxValue should probably be the abs maxValue

Reply:
Fixed with 16465e65